### PR TITLE
Fix gitlab (and IDEA) build

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**

### CODE REVIEW

**Review by kubawach — APPROVED** [review]
Ouch!

**Comment by iNikem** [comment]
Just tried to run it from my IDEA. It failed :D

**Comment by mateuszrzeszutek** [comment]
> Just tried to run it from my IDEA. It failed :D

It passes when you run it from console 🙈

**Comment by mateuszrzeszutek** [comment]
It fails in IDEA for me too, let me fix that thing

File: testing/agent-test-extension/src/main/java/com/splunk/opentelemetry/testing/IgnoredTestTypesConfigurer.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesBuilder;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesConfigurer;
+
+@AutoService(IgnoredTypesConfigurer.class)
+public class IgnoredTestTypesConfigurer implements IgnoredTypesConfigurer {
+  @Override
+  public void configure(Config config, IgnoredTypesBuilder builder) {
+    // we don't want to instrument auto-generated mocks
+    builder.ignoreClass("org.mockito");
```
Comment (mateuszrzeszutek) [inline testing/agent-test-extension/src/main/java/com/splunk/opentelemetry/testing/IgnoredTestTypesConfigurer.java:29]: I've double-checked all my latest changes in the upstream, mockito was NEVER ignored... I have no idea how this worked up till now 🤷

**Pull request merged**